config/config.yml

```diff
@@ -17,7 +17,8 @@ default: &default
       authentication: plain
       enable_starttls_auto: false
     system_mailer_receiver: noreply@beacon-os.com
-    create_test_app_on_new_account: true
+    redis_url: redis://localhost:6379
```

Comment:
this will be overridden by the env variable config, right?

Reply:
@taliaga Yes